client/src/AjaxHelper.js

```diff
@@ -2,7 +2,7 @@ import $ from "jquery";
 
 class AjaxHelperInstance {
     constructor() {
-        this.url = null;
+        this.url = "http://localhost:9000"; // TODO: Config
```

Comment:
If we're initializing this here, should we remove the assignment in index.js for now so we don't have this TODO in two places?

Reply:
Made change